docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -947,6 +946,13 @@ public class CustomLoadBalancerConfiguration {
 
 NOTE:: `HealthCheckServiceInstanceListSupplier` has its own caching mechanism based on Reactor Flux `replay()`, therefore, if it's being used, you may want to skip wrapping that supplier with `CachingServiceInstanceListSupplier`.
 
+[[spring-cloud-loadbalancer-hints]]
+=== Spring Cloud LoadBalancer Hints
+
+Spring Cloud LoadBalancer allows you to set `String` hints that will be passed to the LoadBalancer within the `Request` object and that can later be used in `ReactiveLoadBalancer` implementations that can handle them.
```

Comment:
Change to 

Spring Cloud LoadBalancer lets you set `String` hints that are passed to the LoadBalancer within the `Request` object and that can later be used in `ReactiveLoadBalancer` implementations that can handle them.

(Your sentence is correct, but we have a different editorial standard.)